enterprise/server/backends/pebble_cache/pebble_cache_test.go

```diff
@@ -3,6 +3,7 @@ package pebble_cache_test
 import (
 	"bytes"
 	"context"
+	crand "crypto/rand"
```

Comment:
please move this down below with the other aliased imports

Reply:
is there a reason we need to have both rands in this file?

Reply:
Some tests use `math/rand.IntN()` which is cleaner with `math/rand`.

I plan to switch most of these to `math/rand/v2` in the future, but for now, I created a smaller helper func to use `crypto/rand` in this file instead.